docs-devsite/vertexai.vertexaibackend.md

```diff
@@ -0,0 +1,58 @@
+Project: /docs/reference/js/_project.yaml
+Book: /docs/reference/_book.yaml
+page_type: reference
+
+{% comment %}
+DO NOT EDIT THIS FILE!
+This is generated by the JS SDK team, and any local changes will be
+overwritten. Changes should be made in the source code at
+https://github.com/firebase/firebase-js-sdk
+{% endcomment %}
+
+# VertexAIBackend class
+Represents the configuration class for the Vertex AI backend. Use this with [AIOptions](./vertexai.aioptions.md#aioptions_interface) when initializing the server with [getAI()](./vertexai.md#getai_a94a413)<!-- -->.
+
+<b>Signature:</b>
+
+```typescript
+export declare class VertexAIBackend extends Backend 
+```
+<b>Extends:</b> [Backend](./vertexai.backend.md#backend_class)
+
+## Constructors
+
+|  Constructor | Modifiers | Description |
+|  --- | --- | --- |
+|  [(constructor)(location)](./vertexai.vertexaibackend.md#vertexaibackendconstructor) |  | Creates a configuration object for the Vertex AI backend. |
+
+## Properties
+
+|  Property | Modifiers | Type | Description |
+|  --- | --- | --- | --- |
+|  [location](./vertexai.vertexaibackend.md#vertexaibackendlocation) |  | string | The region identifier. See [Vertex AI locations](https://firebase.google.com/docs/vertex-ai/locations?platform=web#available-locations) for a list of supported locations. |
+
+## VertexAIBackend.(constructor)
+
+Creates a configuration object for the Vertex AI backend.
+
+<b>Signature:</b>
+
+```typescript
+constructor(location?: string);
+```
+
+#### Parameters
+
+|  Parameter | Type | Description |
+|  --- | --- | --- |
+|  location | string | The region identifier, defaulting to <code>us-central1</code>; see [Vertex AI locations](https://firebase.google.com/docs/vertex-ai/locations?platform=web#available-locations) for a list of supported locations. |
```

Comment:
```suggestion
|  location | string | The region identifier, defaulting to <code>us-central1</code>; see [Vertex AI locations](https://firebase.google.com/docs/vertex-ai/locations#available-locations) for a list of supported locations. |
```

Reply:
We've shifted away from the orange buttons for lang/plat in the dev docs.  This link wouldn't have broken, though.

Reply:
done